recipes/juypterlab-language-packs/meta.yaml

```
@@ -0,0 +1,123 @@
+    sha256: c045a15d11d5b0793c8d9ef056c97926dd4e276a11770cd0d84267cdcd45bd67
+  - folder: ja-JP
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}-ja-JP/{{ name }}-ja-JP-{{ version }}.tar.gz
+    sha256: 3a459038d4618e8808b121ced55c4b0fa2b0ba6866a43c9ae548c2a4ca8795c3
+  - folder: ko-KR
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}-ko-KR/{{ name }}-ko-KR-{{ version }}.tar.gz
+    sha256: 71c803f5995363b92b17c90686279c395b2ad19ea60fc88e57a96a4ffa395888
+  - folder: nl-NL
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}-nl-NL/{{ name }}-nl-NL-{{ version }}.tar.gz
+    sha256: 0b44dc3c02b4cb1a3de993bfa9195751e9134ff9f7b8bed056c2d59bc97e02b3
+  - folder: no-NO
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}-no-NO/{{ name }}-no-NO-{{ version }}.tar.gz
+    sha256: f98470bd936302cb57ea8117ce1a50eabe068bafbc6ed7192edfadb673a01a7d
+  - folder: pl-PL
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}-pl-PL/{{ name }}-pl-PL-{{ version }}.tar.gz
+    sha256: cfb1b6b8ce5aa8cd87dfd351484dfe3137ff895f08edc81790b7cf74d6dd32cb
+  - folder: pt-BR
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}-pt-BR/{{ name }}-pt-BR-{{ version }}.tar.gz
+    sha256: 046c098375b0f8cc0bf9afa5768ca4f9c921ff4097c921324d60884fda8659ee
+  - folder: ro-RO
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}-ro-RO/{{ name }}-ro-RO-{{ version }}.tar.gz
+    sha256: 41a7d2f63a98875395392786e4ee78cfad613507bbbf30ee7478ddd880eea304
+  - folder: ru-RU
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}-ru-RU/{{ name }}-ru-RU-{{ version }}.tar.gz
+    sha256: 50c2dd12dc5547bbf9b0269b224f5c9459e3563a455f4d304a1d17696b8aabf2
+  - folder: tr-TR
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}-tr-TR/{{ name }}-tr-TR-{{ version }}.tar.gz
+    sha256: d19789802e7ac12a572af7548664a464b3b45412b68160edf3f4b24c433f5b0f
+  - folder: uk-UA
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}-uk-UA/{{ name }}-uk-UA-{{ version }}.tar.gz
+    sha256: ce4c548ddbb1566bb3ea258abd2f08775079071b7ff8f4e3cbb51db06e750f22
+  - folder: vi-VN
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}-vi-VN/{{ name }}-vi-VN-{{ version }}.tar.gz
+    sha256: 9fd3176c13051a5d8c6c7998189b23343e3e248f1fe58803c95976643e9349e1
+  - folder: zh-CN
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}-zh-CN/{{ name }}-zh-CN-{{ version }}.tar.gz
+    sha256: cef77cc2fa9d4a56fbfcd47ee344c80fb807e3abdf94ef7d80312713a607a067
+  - folder: zh-TW
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}-zh-TW/{{ name }}-zh-TW-{{ version }}.tar.gz
+    sha256: 8b96f81306be0806ed92d3b6f9bec219c79e76b8c1b99deb56bb1fcf3028a6ec
+
+build:
+  number: 0
+  noarch: python
+  script: cd {{ locale }} && {{ PYTHON }} -m pip install . -vv --no-deps
```

This should also work
```suggestion
  script: {{ PYTHON }} -m pip install {{ locale }} -vv --no-deps
```